### PR TITLE
Fix: Kit-library - UI fixes [ED-3840]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: core/app/modules/kit-library/assets/js/pages/index/index.js [hunk 1]
```diff
@@ -228,26 +229,16 @@ export default function Index( props ) {
 							isSuccess && 0 === data.length && queryParams.ready && props.renderNoResultsComponent( {
 								defaultComponent: <ErrorScreen
 									title={ __( 'No results matched your search.', 'elementor' ) }
-									description={ __( 'Try different keywords or continue browsing.', 'elementor' ) }
+									description={ __( 'Try different keywords or ', 'elementor' ) }
```
Comment (StyleShit) [inline core/app/modules/kit-library/assets/js/pages/index/index.js:232]: Why?

File: core/app/modules/kit-library/assets/js/components/bottom-promotion.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Text, Button } from '@elementor/app-ui';
+
+import './bottom-promotion.scss';
+
+export default function BottomPromotion() {
```
Comment (StyleShit) [inline core/app/modules/kit-library/assets/js/components/bottom-promotion.js:5]: Maybe rename it to `EnvatoPromotion`?
Unless there is no way we're gonna have another promotion at the bottom

**Review by StyleShit — CHANGES_REQUESTED** [review]

**Comment by StyleShit** [comment]
Something looks off with the ScreenShotter recently... @matipojo

**Review by matipojo — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please pull from develop.

File: core/app/modules/kit-library/assets/js/pages/index/index.js (hunk 1 above)
Comment (Nevoss) [reply core/app/modules/kit-library/assets/js/pages/index/index.js:232]: The 'continue browsing' is a button next to the description.

File: core/app/modules/kit-library/assets/js/components/bottom-promotion.js (hunk 2 above)
Comment (Nevoss) [reply core/app/modules/kit-library/assets/js/components/bottom-promotion.js:5]: You right, Done.

File: core/app/modules/kit-library/assets/js/components/filter-indication-text.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ export default function FilterIndicationText( props ) {
 	return (
 		<Grid container className="e-kit-library__filter-indication">
 			<Text className="e-kit-library__filter-indication-text">
-				{ sprintf( __( 'Showing %s Results For', 'elementor' ), props.resultCount ) }
+				{ sprintf( _n( 'Showing %s result for', 'Showing %s results for', props.resultCount, 'elementor' ), ! props.resultCount ? __( 'no', 'elementor' ) : props.resultCount ) }
```
[the remark this answers is not in the document]
Comment (Nevoss) [reply core/app/modules/kit-library/assets/js/components/filter-indication-text.js:14]: This was a specific request from "product" to use capital letters only in some cases.

**Review by StyleShit — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**